### ISSUE TITLE
Tooltip layers do not work if layer CRS is not WGS84

### ISSUE THREAD

**Issue by gioman** [3liz/lizmap-web-client#2221]
This seems a re-occurrence of https://github.com/3liz/lizmap-web-client/issues/895

If have tested on the latest LMWC 3.3 and 3.4 on QGIS Server 3.16 on a Ubuntu 20.04 server.

The "Tooltip layers" functionality seems to work fine only if the layer CRS is 4326, regardless of the project CRS.

The same identical layer saved in the same format (tested GPKG) and in a projected CRS (tested 3763) will cause the functionality to now work as expected: the tooltip is not not shown, and if the tool is configured to show geometries in some color, then hovering the mouse of the feature will cause them to change color, but then this is not removed from the canvas when the mouse move over a different feature.

**Comment by gioman** [comment]
Adding here a sample project + data as I cannot provide a public URL

[sample_project.zip](https://github.com/3liz/lizmap-web-client/files/6228208/sample_project.zip)

**Comment by gioman** [comment]
I have seen again this problem recently, using QGIS Server LTR on a Ubuntu 20.04 machine and the latest and greatest version of LWC.

At some point while configuring the project with the LM plugin the tooltip functionality stopped to work and behave as described. Also the "locate" functionality started to point/zoom to the same (wrong) area.

I wiped the CFG file and started over, and all was working again.

I saved a copy of the offending CFG file, that I attach here for analysis.

[projecto1.qgs - Copy.cfg.zip](https://github.com/3liz/lizmap-web-client/files/6771850/projecto1.qgs.-.Copy.cfg.zip)
